src/engraving/rendering/score/harmonylayout.h

```diff
@@ -32,6 +31,33 @@ class System;
 }
 
 namespace mu::engraving::rendering::score {
+// Help class.
+// Contains harmonies/fretboard per segment.
+class HarmonyList : public std::vector<EngravingItem*>
```

Comment:
Given that it's not used in the .h file, this declaration can be in the .cpp file as well. (The only reasons to put something in a .h file are either that it's necessary, or to expose it to other files.)